api/sample.env

```
@@ -111,8 +107,8 @@ AIRTABLE_API_KEY=
 # example: app5JZ0**********
 AIRTABLE_BASE=
```

Reprendre le vocabulaire Airtable pour que la variable ne  soit pas ambigüe
https://blog.airtable.com/what-is-a-database/
```suggestion
AIRTABLE_DATABASE_ID=
```

---

On compte pas forcément le traiter dans cette PR pour ne pas l'alourdir, les impacts sont multiples dans le code et cette variable date d'avant la PR.